workflows/fetch_data.py

```
@@ -0,0 +1,52 @@
+import subprocess
+import httpx
+import polars as pl
+import pyarrow.ipc as ipc
+from io import BytesIO
+
+from google.cloud import run_v2
```

Pylint (3.3.7)

[error] No name 'run_v2' in module 'google.cloud'

(E0611)